server/src/main/resources/application.yml

```diff
@@ -64,4 +47,22 @@ micronaut:
       login-success-target-url: '/'
       unauthorized-target-url: '/unauthorized'
       forbidden-target-url: '/forbidden'
-      
\ No newline at end of file
+---      
+datasources:
+  default:
+    driver-class-name: org.postgresql.Driver
+    dialect: POSTGRES
```

Comment:
we don't need to pass username and password ?

Reply:
It should be picked up from the ENV, I believe. I may have to change the env var passed to DATASOURCES_DEFAULT_PASSWORD and DATASOURCES_DEFAULT_USERNAME, but I'll see on deploy.

Reply:
Just altered this...pretty positive it was going to cause an error.